nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -634,13 +634,14 @@ public static void compareGenericTypeParameterNullabilityForCall(
       }
       if (enclosingType != null) {
         invokedMethodType =
-            TypeSubstitutionUtils.memberType(state.getTypes(), enclosingType, methodSymbol);
+            TypeSubstitutionUtils.memberType(state.getTypes(), enclosingType, methodSymbol, config);
```

Comment:
Is the passing around of the config variable everywhere required for this bug fix? I don't see it being used anywhere.

Or is it an unrelated change which has been combined with this PR? If so, can we clarify this in the PR description?

Reply:
The `Config` object is required for this bug fix.  It is stored in an instance field of `RestoreNullnessAnnotationsVisitor` and used to check whether an annotation should be treated as `@Nullable` or `@NonNull` here:

https://github.com/uber/NullAway/pull/1143/files#diff-a6bb8226bed276672e56b162bca51775aa7ad9c2d3fc69c787bc96e36ce33507R137-R138